psrm/reports/dr.py

```diff
@@ -0,0 +1,97 @@
+import psrm
+from psrm.enums.fordringshaver import FordringsHaver as FH
+from tqdm import tqdm
+from report_utils import *
+
+
+# Load data
+ps = psrm.psrm_utils.cache_psrm(path=psrm.path_v4, input=psrm.v4)
+acl = annotate_acl(ps.udtraeksdata, claimant=FH.DR.value['Claimant_ID'])
+udl = annotate_udl(ps.udligning)
+afr = annotate_afr(ps.underretning)
+dr = saldo_DR()
+rm = saldo_PSRM()
+
+# Calculate totals of 'underretninger' on HF/~HF per NYMFID
+afr_hf = sum_NYMF(afr).rename(columns={'AMOUNT': 'AFR_HF'})
+afr_ir = sum_NYMF(afr, HF=False).rename(columns={'AMOUNT': 'AFR_IR'})
+udl_hf = sum_NYMF(udl).rename(columns={'AMOUNT': 'UDL_HF'})
+udl_ir = sum_NYMF(udl, HF=False).rename(columns={'AMOUNT': 'UDL_IR'})
+
+# Calculate debt, payments and interests per NYMFID
+debt = acl_debt(acl)
+pays = acl_pay(acl)
+intr = acl_interest(acl)
+
+# Create report, merge debt
+rep = pd.DataFrame(acl['NYMFID'].unique(), columns=['NYMFID'])
+rep = pd.merge(rep, debt, on='NYMFID', how='left')
+rep = pd.merge(rep, pays, on='NYMFID', how='left')
+rep = pd.merge(rep, intr, on='NYMFID', how='left')
+
+# Merge totals to report
+rep = pd.merge(rep, afr_hf, on='NYMFID', how='left')
+rep = pd.merge(rep, afr_ir, on='NYMFID', how='left')
+rep = pd.merge(rep, udl_hf, on='NYMFID', how='left')
+rep = pd.merge(rep, udl_ir, on='NYMFID', how='left')
+rep = rep.fillna(0)
+
+# Calculate remaining debt per NYMFID
+rep['DEBT_REMAIN'] = (rep['DEBT']-rep['PAYMENT']+rep['ACL_IR']).round(2)
+rep['ACL_HF'] = (rep['PAYMENT'] - rep['ACL_IR']).round(2)
+rep = pd.merge(rep, rm, on='NYMFID', how='left')
+assert rep['NYMFID'].is_unique
```

Comment:
Jeg har det lidt mærkeligt med asserts når det ikke er i en test. Hvad med noget i stil med:
```suggestion
if not rep['NYMFID'].is_unique:
    raise SomeError("Der er ikke-unikke NyMF ID'er")
```
Jeg tror jeg har en `NyMFError` implementeret.

Reply:
Jeg benytter NyMFError istedet.